plugin-server/src/main/ingestion-queues/session-recording/session-recordings-blob-consumer.ts

```diff
@@ -2,6 +2,7 @@ import * as Sentry from '@sentry/node'
 import { captureException } from '@sentry/node'
 import { mkdirSync, rmSync } from 'node:fs'
 import { CODES, HighLevelProducer as RdKafkaProducer, Message, TopicPartition } from 'node-rdkafka-acosom'
+import Kafka from 'node-rdkafka-acosom'
```

Comment:
Not sure want the Pinter said but this could also be imported the line above.

Reply:
Now I want to write a Harold Pinter themed Linter 🤣